### PR TITLE
[Keyboard] add rart75

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

re add new keyboard rart75 from #9410 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/rart/rart75/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# [RART 75](https://github.com/alabahuy/RART/tree/master/RART75)
+![RART 75 TOP](https://user-images.githubusercontent.com/30220306/85908009-9de88a80-b83d-11ea-84e8-d1bc07e6bf38.png)
+75% pcb with encoder, based on IMKG (Indonesia Mechanical Keyboard Group)
```
Comment (zvecr) [inline keyboards/rart/rart75/readme.md:3]:
```suggestion
# [RART 75](https://github.com/alabahuy/RART/tree/master/RART75)

![RART 75 TOP](https://user-images.githubusercontent.com/30220306/85908009-9de88a80-b83d-11ea-84e8-d1bc07e6bf38.png)

75% pcb with encoder, based on IMKG (Indonesia Mechanical Keyboard Group)
```

File: keyboards/rart/rart75/config.h [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+#define MATRIX_ROW_PINS { F1, F4, F6, C7, D4, D0 }
+#define MATRIX_COL_PINS { D5, D3, D2, D1, C6, B6, B5, B4, D7, D6, B3, B1, F7, F5, B2, B7 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* number of backlight levels */
+#define BACKLIGHT_LEVELS 0
+
```
Comment (zvecr) [inline keyboards/rart/rart75/config.h:44]:
```suggestion
```

Comment (alabahuy) [reply keyboards/rart/rart75/config.h]: done @zvecr

File: keyboards/rart/rart75/rules.mk [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
```
Comment (fauxpark) [inline keyboards/rart/rart75/rules.mk:30]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
```

Comment (alabahuy) [reply keyboards/rart/rart75/rules.mk]: done @fauxpark

**Comment by alabahuy** [comment]
@fauxpark @zvecr can it  beapproved?

**Comment by alabahuy** [comment]
@zvecr can it be approved?

File: keyboards/rart/rart75/readme.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# [RART 75](https://github.com/alabahuy/RART/tree/master/RART75)
+
+![RART 75 TOP](https://user-images.githubusercontent.com/30220306/85908009-9de88a80-b83d-11ea-84e8-d1bc07e6bf38.png)
+
+75% pcb with encoder, based on IMKG (Indonesia Mechanical Keyboard Group)
+* Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)
```
Comment (fauxpark) [inline keyboards/rart/rart75/readme.md:6]:
```suggestion

* Keyboard Maintainer: [Alabahuy](https://github.com/alabahuy)
```

Comment (alabahuy) [reply keyboards/rart/rart75/readme.md:7]: done @fauxpark

**Comment by alabahuy** [comment]
@zvecr can it be merged?

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**